lib/data/wrapper.js

```diff
@@ -136,6 +136,16 @@ const data = {
     // It would be preferable to have an sproxyd batch delete route to
     // replace this
     batchDelete: (locations, log) => {
+        // TODO: The method of persistence of sproxy delete key will
+        // be finalized; refer Issue #312 for the discussion. In the
+        // meantime, we at least log the location of the data we are
+        // about to delete before attempting its deletion.
+        log.trace('About to initiate batchDelete, logging object keys in ' +
```

Comment:
We should aim for shorter log messages, as it would be easier to grep or to look at in Kibana. 
`initiating batch delete` should be enough IMO. You can extend the comment mentioning gc to add some context.
Edit: We should be careful of logging large objects as this may contribute to negative performance.